src/pages/services/aem-cf-console-admin/api/index.md

```diff
@@ -81,45 +84,75 @@ const guestConnection = await register({
     id: "my.company.extension-with-header-menu-button",
     methods: {
         headerMenu: {
-            getButton() {
-                return {
-                    id: "import",
-                    label: "Import",
-                    icon: 'OpenIn',
-                };
+            getButtons() {
+                return [
+                    {
+                        id: "my.company.export-button",
+                        label: "Export",
+                        icon: 'Export', // Spectrum workflow icon code from https://spectrum.adobe.com/page/icons/
+                        subItems: [
+                            {
+                                id: 'xml',
+                                label: 'XML',
+                                onClick: async () => {
+                                    console.log('Export in XML button has been pressed.');
+                                    // ...
+                                },
+                            },
+                            {
+                                id: 'csv',
+                                label: 'CSV',
+                                onClick: async () => {
+                                    console.log('Export in CSV button has been pressed.');
+                                    // ...
+                                },
+                            },
+                        ],
+                    },
+                ];
             },
-            onClick() {
-                console.log('Import button has been pressed');
-            }
         },
-    }
+    },
 });
 ```
 
-The callback is invoked when user clicks on the button and does not receive any extra parameters.
+The `onClick` callback is invoked when user clicks on the button and does not receive any extra parameters.
```

Comment:
```suggestion
The `onClick` callback is invoked when a user clicks on the button. It does not receive any arguments.
```

Reply:
It's a small thing: "parameters" are what the function signature defines, "arguments" are the values received. I thought this was clearer, but it's just a suggestion.

Reply:
done